spread.yaml

```diff
@@ -76,6 +76,9 @@ backends:
             - opensuse-42.3-64:
                 image: opensuse-cloud/opensuse-leap-42-3-v20180116
                 workers: 4
+            - arch-64:
```

Comment:
you should use arch-linux-64 instead of arch-64 and then you don't need to set the image. So the only var you are going to define is the workers.
The idea is to avoid setting the image so when we update the images it is going to automatically run with the newest one.

Reply:
I've pushed a change that updates the name to `arch-linux-64`. If it's green, I'll proceed with the rest of the updates.